content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -1,146 +1,404 @@
+Settings`. Scroll to the :guilabel:`Pricing` section, and click the checkbox next to the
+:guilabel:`Pricelists` feature.
+
+Doing so reveals two additional options beneath it: :guilabel:`Multiple prices per product` and
+:guilabel:`Advanced price rules (discounts, formulas)`. A link labeled :guilabel:`Pricelists` also
+appears, which reveals a separate pricelists page, wherein pricelists can be created and/or
+modified.
+
+.. image:: pricing/pricelist-feature-setting.png
    :align: center
+   :alt: How the pricelist feature setting looks in Odoo Sales.
 
-.. note::
-   * *A single sale price per product:* doesn't let you adapt prices, it use default product
-     price ;
-   * *Different prices per customer segment:* you will set several prices per products ;
-   * *Advanced pricing based on formula:* will let you apply discounts, margins and roundings.
+- :guilabel:`Multiple prices per product`: provides the option to set several prices per product.
+- :guilabel:`Advanced price rules (discounts, formulas)`: provides the option to create detailed
+  price rules and apply discounts, margins, and roundings.
 
-Several prices per product
-==========================
+After clicking the checkbox beside the :guilabel:`Pricelists` feature, select one of those two
+options, then click :guilabel:`Save` to save all changes.
 
-To apply several prices per product, select *Different prices per customer
-segment* in :menuselection:`Sales --> Settings`. Then open the *Sales* tab
-in the product detail form. You can settle following strategies.
+Pricelists
+==========
 
-Prices per customer segment
----------------------------
+After activating and saving the :guilabel:`Pricelists` feature, the :guilabel:`Settings` page
+reloads and, from here, either select the :guilabel:`Pricelists` link (beneath the
+:guilabel:`Pricelists` feature on the :guilabel:`Settings` page), or navigate to
+:menuselection:`Sales app --> Products --> Pricelists`.
 
-Create pricelists for your customer segments: e.g. registered, premium, etc.
+Either option reveals a separate :guilabel:`Pricelists` page, in which pricelists can be created
+and/or modified at any time.
 
-.. image:: pricing/pricing_customer.png
+.. image:: pricing/pricelists-page.png
    :align: center
+   :alt: How the pricelists page looks in Odoo Sales.
+
+.. note::
+   The :guilabel:`Public Pricelist` is the default pricelist used with Odoo *eCommerce*.
+
+.. tip::
+   The pricelists sequence can be changed by dragging and dropping pricelists in any order, while
+   in list view.
 
-The default pricelist applied to any new customer is *Public Pricelist*. To
-segment your customers, open the customer detail form and change the *Sale
-Pricelist* in the *Sales & Purchases* tab.
+From the :guilabel:`Pricelists` page, either select the desired pricelist to edit, or click
+:guilabel:`New` to create a new pricelist, which reveals a blank pricelist detail form that can be
```

Comment:
Understood. I changed the majority of the portions where "detail form" was used - however, I kept a few in, mainly for when describing a "Customer detail form" - mostly because saying a "customer's customer form" didn't sound great...haha - but, if you think it's necessary to remove those, as well, just let me know.